indexer/feature_meta.cpp

```diff
@@ -1,10 +1,18 @@
 #include "indexer/feature_meta.hpp"
 
 #include "std/algorithm.hpp"
+#include "std/target_os.hpp"
 
 namespace feature
 {
 
+static char constexpr const * kBaseWikiUrl =
```

Comment:
Do not use static, use anonymous namespaces instead. static for translation-unit-globals are quite unusual in our code.